sections/slideshow.liquid

```diff
@@ -338,6 +347,26 @@
       "label": "t:sections.slideshow.settings.change_slides_speed.label",
       "default": 5
     },
+    {
+      "type": "header",
+      "content": "t:sections.slideshow.settings.animations.content"
```

Comment:
This is pointing to an invalid translation:
<img width="248" alt="image" src="https://user-images.githubusercontent.com/37168033/226391631-ed17b41b-282a-43fe-83fe-b2f0529cc806.png">

I think you meant 
```suggestion
      "content": "t:sections.all.animation.content"
```